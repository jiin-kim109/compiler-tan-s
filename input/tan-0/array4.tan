main {
	const a := new [bool](3);
	[a:0] := true;
	[a:1] := true;
	print a \n;
	var b := new [char](2);
	print b \n;
	var c := new [string](3);
	[c:0] := "yes";
	print c \n;
}