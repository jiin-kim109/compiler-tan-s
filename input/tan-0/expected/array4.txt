[true,true,false]
[,]
[yes,è,è]
